nodes/nomos-node/src/lib.rs

```diff
@@ -1,225 +1,184 @@
-    cryptarchia_consensus::network::adapters::libp2p::LibP2pAdapter<Tx, BlobInfo>,
-    cryptarchia_consensus::blend::adapters::libp2p::LibP2pAdapter<
-        BlendNetworkAdapter,
-        Tx,
-        BlobInfo,
-    >,
-    MockPool<HeaderId, Tx, <Tx as Transaction>::Hash>,
-    MempoolNetworkAdapter<Tx, <Tx as Transaction>::Hash>,
-    MockPool<HeaderId, BlobInfo, <BlobInfo as DispersedBlobInfo>::BlobId>,
-    MempoolNetworkAdapter<BlobInfo, <BlobInfo as DispersedBlobInfo>::BlobId>,
-    FillSizeWithTx<MB16, Tx>,
-    FillSizeWithBlobs<MB16, BlobInfo>,
-    RocksBackend<Wire>,
-    KzgrsSamplingBackend<ChaCha20Rng>,
-    SamplingAdapter,
-    ChaCha20Rng,
-    SamplingStorageAdapter<DaShare, Wire>,
-    KzgrsDaVerifier,
-    VerifierNetworkAdapter<NomosDaMembership>,
-    VerifierStorageAdapter<DaShare, Wire>,
-    SystemTimeBackend,
-    HttApiAdapter<NomosDaMembership>,
->;
-
-pub type NodeDaIndexer =
-    DaIndexer<nomos_da_sampling::network::adapters::validator::Libp2pAdapter<NomosDaMembership>>;
-
-pub type DaSampling<SamplingAdapter> = DaSamplingService<
-    KzgrsSamplingBackend<ChaCha20Rng>,
-    SamplingAdapter,
-    ChaCha20Rng,
-    SamplingStorageAdapter<DaShare, Wire>,
-    KzgrsDaVerifier,
-    VerifierNetworkAdapter<NomosDaMembership>,
-    VerifierStorageAdapter<DaShare, Wire>,
-    HttApiAdapter<NomosDaMembership>,
->;
-
-pub type NodeDaSampling = DaSampling<SamplingLibp2pAdapter<NomosDaMembership>>;
-
-pub type DaVerifier<VerifierAdapter> =
-    DaVerifierService<KzgrsDaVerifier, VerifierAdapter, VerifierStorageAdapter<DaShare, Wire>>;
-
-pub type NodeDaVerifier = DaVerifier<VerifierNetworkAdapter<FillFromNodeList>>;
-
-pub type NomosTimeService = TimeService<SystemTimeBackend>;
-
-#[derive(Services)]
+#[derive_services]
 pub struct Nomos {
     #[cfg(feature = "tracing")]
-    tracing: OpaqueServiceHandle<Tracing>,
-    network: OpaqueServiceHandle<NetworkService<NetworkBackend>>,
-    blend: OpaqueServiceHandle<BlendService<BlendBackend, BlendNetworkAdapter>>,
-    da_indexer: OpaqueServiceHandle<NodeDaIndexer>,
-    da_verifier: OpaqueServiceHandle<NodeDaVerifier>,
-    da_sampling: OpaqueServiceHandle<NodeDaSampling>,
-    da_network: OpaqueServiceHandle<DaNetworkService<DaNetworkValidatorBackend<NomosDaMembership>>>,
-    cl_mempool: OpaqueServiceHandle<TxMempool>,
-    da_mempool: OpaqueServiceHandle<DaMempool>,
-    cryptarchia: OpaqueServiceHandle<NodeCryptarchia>,
-    time: OpaqueServiceHandle<NomosTimeService>,
-    http: OpaqueServiceHandle<NomosApiService>,
-    storage: OpaqueServiceHandle<StorageService<RocksBackend<Wire>>>,
-    system_sig: OpaqueServiceHandle<SystemSig>,
-}
-
-pub struct Wire;
-
-impl StorageSerde for Wire {
-    type Error = wire::Error;
-
-    fn serialize<T: Serialize>(value: T) -> Bytes {
-        wire::serialize(&value).unwrap().into()
-    }
-
-    fn deserialize<T: DeserializeOwned>(buff: Bytes) -> Result<T, Self::Error> {
-        wire::deserialize(&buff)
-    }
```

Comment:
Where did the wire stuff go?

Reply:
It's still there, just higher up in the module.